controllers/dspipeline_controller.go

```diff
@@ -190,12 +188,26 @@ func (r *DSPAReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 
 	defer r.updateStatus(ctx, dspa, dspaStatus, log, req)
 
+	if dspa.Spec.DSPVersion != config.DSPV2VersionString {
+		err1 := fmt.Errorf("unsupported DSP version %s detected. Please manually remove "+
```

Comment:
Could this message include a link to the Release Notes _Support Removals_ section or to a KCS article about the removal of DSPv1?

Also, I think "unsupported" should start with capital letter (Unsupported)

Reply:
> Could this message include a link to the Release Notes Support Removals section or to a KCS article about the removal of DSPv1?

I understand the need for this, but we'd be adding rhoai links in the odh repo, which I think we want to avoid for now. I'm not fully convinced we shouldn't do this, but let's wait until we have the KCS article first, maybe we can do a follow up pr. 

> Also, I think "unsupported" should start with capital letter (Unsupported)

the convention in golang is to start error messages with a lowercase